proto-lens-discrimination/proto-lens-discrimination.cabal

```diff
@@ -0,0 +1,34 @@
+-- Initial proto-lens-discrimination.cabal generated by cabal init.  For 
+-- further documentation, see http://haskell.org/cabal/users-guide/
+
+name:                proto-lens-discrimination
+version:             0.1.0.0
+synopsis:            Support for using proto-lens types with http://hackage.haskell.org/package/discrimination.
+description:         Reflection-based implementations of Sort for proto-lens' generated types.
+
+homepage:            https://github.com/google/proto-lens
+license:             BSD3
+license-file:        LICENSE
+author:              Andrew Pritchard
+maintainer:          awpr+protolens@google.com
+copyright:           Google Inc.
+category:            Data
+build-type:          Simple
+cabal-version:       >=1.10
+
+library
+  exposed-modules:     Data.ProtoLens.Sort
+                     , Data.ProtoLens.Group
+                     , Data.ProtoLens.Discrimination
+  build-depends:       base >=4.8 && <4.9
+                     , data-default >=0.5 && <0.6
+                     , bytestring >=0.10.6
```

Comment:
bytestring==0.10.* (similar to proto-lens itself)
It looks like you're not using anything fancy in proto-lens itself; also we're giving upper bounds to all the other packages.